### DISCUSSION TITLE
What company launched "Olive oil-infused coffee" in Italy recently?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#183] [Galaxy Trivia] [answered]
**Hint:** 🧜‍♀️

**Comment by thenithinbalaji** [comment, marked as answer]
The American multinational chain of coffeehouses, Starbucks Corporation has lately launched a line of olive oil-infused drinks in Italy. Howard Schultz, CEO of Starbucks in a statement said that the “unexpected, velvety, buttery flavour… enhanced the coffee and lingers beautifully on the palate.”
